### PR TITLE
README updates for the main repo, and a light overview on the docs subdir

### PR DESCRIPTION
**Pull request by akleiner2**
**Overview**
Updating some links now that the new evergreen.segment.com is live. 

**Screenshots (if applicable)**
N/A

**Documentation**
- [ ] ~Updated Typescript types and/or component PropTypes~
- [ ] ~Added / modified component docs~
- [ ] ~Added / modified Storybook stories~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,4 +213,4 @@ Please respect our [Code of Conduct](.github/CODE_OF_CONDUCT.md), in short:
 Evergreen is released under the MIT license.
 The BlueprintJS icons are licensed under a [custom Apache 2.0 license](https://github.com/palantir/blueprint/blob/develop/LICENSE).
 
-Copyright © 2017 Segment.io, Inc.
+Copyright © 2021 Segment.io, Inc.
```
Comment (kamebkj) [inline README.md:216]: 💯

File: docs/README.md [hunk 2]
```diff
@@ -1,41 +1,25 @@
...
 
-```
-npm install --save-dev @types/react @types/react-dom @types/node
-```
+# Adding a new documentation page
+
+Add an entry under the corresponding header in `IA.ts` - you can think of this as the "database" for all status relating to Evergreen components and documentation.
 
-When we run `next dev` the next time, Next.js will start looking for any `.ts` or `.tsx` files in our project and builds it. It even automatically creates a `tsconfig.json` file for our project with the recommended settings.
+From there, the directory structure in `docs/documentation/` contains the `.mdx` files which mirror the route structure for the docs site.
```
Comment (kamebkj) [inline docs/README.md:20]: Wonder if we also want to call out adding thumbnail images and add them to `public`

**Review by kamebkj — APPROVED** [review]
only one minor call out on thumbnail image. otherwise lgtm!

File: docs/README.md (hunk 2 above)
Comment (akleiner2) [reply docs/README.md:20]: Done! We can always come back here and just link to the `next.js` documentation. We're not doing anything fancy other than using the standard filesystem setup.

**Comment by akleiner2** [comment]
Need to figure out why the actions keep getting cancelled, but given that this is a documentation change, it should be fine.

**Pull request merged**